ksqldb-engine/src/main/java/io/confluent/ksql/function/udf/math/Greatest.java

```diff
@@ -44,7 +44,7 @@
   @Udf
   public Integer greatest(@UdfParameter final Integer val, @UdfParameter final Integer... vals) {
 
-    return Stream.concat(Stream.of(val), Arrays.stream(vals))
+    return (vals == null) ? null : Stream.concat(Stream.of(val), Arrays.stream(vals))
```

Comment:
Here and below, similar to JsonConcat, can `vals` actually be null?

Reply:
Yeah, the only thing left to figure out is how to get a null list of a numeric type.  I'm having a little trouble doing that quickly; that may just be a limitation of my SQL creativity.:)